server/foodAPI/SpoonacularAPI/SpoonacularFoodAPI.ts

```diff
@@ -182,10 +182,10 @@ export default class SpoonacularFoodAPI extends SpoonacularAPI implements IFoodA
         });
 
         // (A && !B) || (!A && B)
```

Comment:
You don't need this comment anymore, but you can add one above line 188 saying `// Checks that if one of these is set, both are set`